baseapp-core/setup.py

```diff
@@ -1,3 +1,7 @@
 from setuptools import setup
 
-setup()
+setup(
+    name="baseapp-core",
+    packages=["baseapp-core"],
+    version="0.1",
```

Comment:
should we keep version from `setup.cfg`? 

```
version = 0.1.5
```

Reply:
I guess we do that once we start using the real account, right now we are on test one